x/gamm/pool-models/balancer/constants.go

```diff
@@ -19,4 +19,5 @@ var (
 	GuaranteedWeightPrecision int64 = 1 << 30
 
 	PoolTypeName string = "Balancer"
+	oneDec              = osmomath.OneDec()
```

Comment:
Just curious, is it worth defining onedec for every big module and spare us heap allocation?

Reply:
The issue is there are no golang constants available for custom types. So we can't trust other modules to not edit it :(